force_gromacs/chemicals/i_fragment.py

```diff
@@ -0,0 +1,20 @@
+from traits.api import Interface, List, Str, Float
+
+
+class IFragment(Interface):
```

Comment:
I very much like this idea of fragments interface. Can we think of any other attributes it should contain?

Reply:
Good question, at the moment I've only included attributes that are required by the `Molecule` class. 

I don't know enough about other MD codes to come up with key shared features of how they represent atomic / molecular ions, so I played it safe with just the 3 listed. If we can think of more, then by all means let's include them